tests/integration/telemetry/stats/prometheus/customizemetrics/customize_metrics_test.go

```diff
@@ -48,26 +48,33 @@ func TestCustomizeMetrics(t *testing.T) {
 	framework.NewTest(t).
 		Features("observability.telemetry.stats.prometheus.customize-metric").
 		Run(func(ctx framework.TestContext) {
-			destinationQuery := buildQuery()
-			var metricVal string
+			httpDestinationQuery, grpcQuery := buildQuery("http"), buildQuery("grpc")
```

Comment:
nit: can you make these two strings const?

Reply:
done